modules/solid_mechanics/include/materials/ReturnMappingModel.h

```diff
@@ -54,6 +54,7 @@ class ReturnMappingModel : public ConstitutiveModel
   const bool _output_iteration_info_on_error;
   const Real _relative_tolerance;
   const Real _absolute_tolerance;
+  Real _effective_strain_increment;
```

Comment:
This new member needs to be initialized in the constructor just so that it can never be retrieved in an uninitialized state.